2021-11-22 00:43:33,483:root:INFO:	Default logging config has been set
2021-11-22 00:43:33,483:__main__:INFO:	Ended initialization
2021-11-22 00:43:33,483:__main__:INFO:	Starting set up
2021-11-22 00:43:33,483:__main__:INFO:	Starting setting configuration
2021-11-22 00:43:33,484:__main__:INFO:	Ended setting configuration
2021-11-22 00:43:33,484:__main__:INFO:	Using default configuration
2021-11-22 00:43:33,488:__main__:INFO:	Ended set up
2021-11-22 00:43:33,488:__main__:INFO:	Starting parsing crontab
2021-11-22 00:43:33,488:__main__:ERROR:	Crontab file path not specified
2021-11-22 00:43:33,488:__main__:INFO:	Finishing cron
2021-11-22 00:43:49,115:root:INFO:	Default logging config has been set
2021-11-22 00:43:49,115:__main__:INFO:	Ended initialization
2021-11-22 00:43:49,116:__main__:INFO:	Starting set up
2021-11-22 00:43:49,116:__main__:INFO:	Starting setting configuration
2021-11-22 00:44:32,396:__main__:INFO:	Ended setting configuration
2021-11-22 00:44:32,396:__main__:INFO:	Using default configuration
2021-11-22 00:44:32,398:__main__:INFO:	Ended set up
2021-11-22 00:44:57,724:__main__:INFO:	Starting parsing crontab
2021-11-22 00:44:57,724:__main__:ERROR:	Crontab file path not specified
2021-11-22 00:44:57,724:__main__:INFO:	Finishing cron
2021-11-22 00:46:26,712:root:INFO:	Default logging config has been set
2021-11-22 00:46:26,713:__main__:INFO:	Ended initialization
2021-11-22 00:46:26,713:__main__:INFO:	Starting set up
2021-11-22 00:46:26,713:__main__:INFO:	Starting setting configuration
2021-11-22 00:46:26,713:__main__:INFO:	Ended setting configuration
2021-11-22 00:46:26,713:__main__:INFO:	Using default configuration
2021-11-22 00:46:26,715:__main__:INFO:	Ended set up
2021-11-22 00:46:26,715:__main__:INFO:	Starting parsing crontab
2021-11-22 00:46:26,715:__main__:ERROR:	Crontab file does not exist by path
./crontab.tab
2021-11-22 00:46:26,715:__main__:INFO:	Finishing cron
2021-11-22 00:47:05,149:root:INFO:	Default logging config has been set
2021-11-22 00:47:05,149:__main__:INFO:	Ended initialization
2021-11-22 00:47:05,149:__main__:INFO:	Starting set up
2021-11-22 00:47:05,149:__main__:INFO:	Starting setting configuration
2021-11-22 00:47:05,149:__main__:INFO:	Ended setting configuration
2021-11-22 00:47:05,149:__main__:INFO:	Using default configuration
2021-11-22 00:47:05,151:__main__:INFO:	Ended set up
2021-11-22 00:47:05,151:__main__:INFO:	Starting parsing crontab
2021-11-22 00:47:05,156:__main__:INFO:	Ended parsing crontab
2021-11-22 00:47:05,156:__main__:INFO:	Starting workflow
2021-11-22 00:47:55,285:root:INFO:	Default logging config has been set
2021-11-22 00:47:55,285:__main__:INFO:	Ended initialization
2021-11-22 00:47:55,285:__main__:INFO:	Starting set up
2021-11-22 00:47:55,285:__main__:INFO:	Starting setting configuration
2021-11-22 00:47:55,286:__main__:INFO:	Ended setting configuration
2021-11-22 00:47:55,286:__main__:INFO:	Using default configuration
2021-11-22 00:47:55,287:__main__:INFO:	Ended set up
2021-11-22 00:47:55,287:__main__:INFO:	Starting parsing crontab
2021-11-22 00:47:55,292:__main__:INFO:	Ended parsing crontab
2021-11-22 00:47:55,292:__main__:INFO:	Starting workflow
2021-11-22 00:48:17,801:__main__:ERROR:	Unknown error
Traceback (most recent call last):
  File "d:/WindBora/Simcord/Cron-python/main.py", line 144, in <module>
    workflow(crontab)
  File "d:/WindBora/Simcord/Cron-python/main.py", line 133, in workflow
    iter_over_crontab(crontab)
  File "d:/WindBora/Simcord/Cron-python/main.py", line 96, in iter_over_crontab
    subprocess.start()
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\process.py", line 121, in start
    self._popen = self._Popen(self)
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\context.py", line 327, in _Popen
    return Popen(process_obj)
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\popen_spawn_win32.py", line 73, in __init__
    hp, ht, pid, tid = _winapi.CreateProcess(
OSError: [WinError 1450] Insufficient system resources exist to complete the requested service
2021-11-22 00:57:38,353:root:INFO:	Default logging config has been set
2021-11-22 00:57:38,353:__main__:INFO:	Ended initialization
2021-11-22 00:57:38,354:__main__:INFO:	Starting set up
2021-11-22 00:57:38,354:__main__:INFO:	Starting setting configuration
2021-11-22 00:57:38,354:__main__:INFO:	Ended setting configuration
2021-11-22 00:57:38,354:__main__:INFO:	Using default configuration
2021-11-22 00:57:38,355:__main__:INFO:	Ended set up
2021-11-22 00:57:38,355:__main__:INFO:	Starting parsing crontab
2021-11-22 00:57:38,357:__main__:INFO:	Ended parsing crontab
2021-11-22 00:57:38,357:__main__:INFO:	Starting workflow
2021-11-22 00:58:21,580:__main__:ERROR:	Unknown error
Traceback (most recent call last):
  File "d:/WindBora/Simcord/Cron-python/main.py", line 148, in <module>
    workflow(crontab)
  File "d:/WindBora/Simcord/Cron-python/main.py", line 137, in workflow
    iter_over_crontab(crontab)
  File "d:/WindBora/Simcord/Cron-python/main.py", line 99, in iter_over_crontab
    subprocess.start()
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\process.py", line 121, in start
    self._popen = self._Popen(self)
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\context.py", line 327, in _Popen
    return Popen(process_obj)
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\popen_spawn_win32.py", line 73, in __init__
    hp, ht, pid, tid = _winapi.CreateProcess(
OSError: [WinError 1455] The paging file is too small for this operation to complete
2021-11-22 00:59:04,578:root:INFO:	Default logging config has been set
2021-11-22 00:59:04,578:__main__:INFO:	Ended initialization
2021-11-22 00:59:04,578:__main__:INFO:	Starting set up
2021-11-22 00:59:04,578:__main__:INFO:	Starting setting configuration
2021-11-22 00:59:04,578:__main__:INFO:	Ended setting configuration
2021-11-22 00:59:04,579:__main__:INFO:	Using default configuration
2021-11-22 00:59:04,580:__main__:INFO:	Ended set up
2021-11-22 00:59:04,580:__main__:INFO:	Starting parsing crontab
2021-11-22 00:59:04,582:__main__:INFO:	Ended parsing crontab
2021-11-22 00:59:04,582:__main__:INFO:	Starting workflow
2021-11-22 00:59:40,741:__main__:ERROR:	Unknown error
Traceback (most recent call last):
  File "d:/WindBora/Simcord/Cron-python/main.py", line 148, in <module>
    workflow(crontab)
  File "d:/WindBora/Simcord/Cron-python/main.py", line 137, in workflow
    iter_over_crontab(crontab)
  File "d:/WindBora/Simcord/Cron-python/main.py", line 99, in iter_over_crontab
    subprocess.start()
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\process.py", line 121, in start
    self._popen = self._Popen(self)
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\context.py", line 327, in _Popen
    return Popen(process_obj)
  File "C:\Users\mc-petry\AppData\Local\Programs\Python\Python38\lib\multiprocessing\popen_spawn_win32.py", line 73, in __init__
    hp, ht, pid, tid = _winapi.CreateProcess(
OSError: [WinError 1455] The paging file is too small for this operation to complete
